core/primitives/src/epoch_manager.rs

```diff
@@ -832,7 +827,19 @@ pub mod epoch_info {
                     shard_cps[(height as u64 % (shard_cps.len() as u64)) as usize]
                 }
                 Self::V3(v3) => {
-                    let seed = {
+                    let protocol_verstion = self.protocol_version();
+                    let seed = if checked_feature!(
+                        "stable",
+                        SynchronizeBlockChunkProduction,
+                        protocol_verstion
```

Comment:
looks like a typo here :)

Reply:
where?

Reply:
fixed it myself :)

Reply:
Ah, I see now, I though you meant there is typo in feature name